src/app/OperationalSessionSetup.h

```diff
@@ -326,6 +327,10 @@ class DLL_EXPORT OperationalSessionSetup : public SessionDelegate,
      * This function will set new IP address, port and MRP retransmission intervals of the device.
      */
     void UpdateDeviceData(const Transport::PeerAddress & addr, const ReliableMessageProtocolConfig & config);
+
+    // This number is taken from AddressResolve.h. Please keep it in sync.
```

Comment:
Why are the constants separated?

There should be a single constant or the comment should explain why that was impossible to do. Why can't app include address_resolve?

Reply:
I spliced the patch as 2 different PRs (the one for operation to come). So I will address that comment there.